controller/internal/routes/identity_router.go

```diff
@@ -278,3 +283,26 @@ func (r *IdentityRouter) getPolicyAdvice(ae *env.AppEnv, rc *response.RequestCon
 	output := MapAdvisorServiceReachabilityToRestEntity(result)
 	rc.RespondWithOk(output, nil)
 }
+
+func (r *IdentityRouter) getPostureData(_ *env.AppEnv, rc *response.RequestContext) {
```

Comment:
why does this function have hard coded values like this? this intentional? if so - can you put a comment in the code as to why? just reviewing this code it seems odd to see hardcoded values here like this.

Reply:
This version is to get the API out for external integration development to start. 

I won't have real posture data until a later commit.